File: 2767-maximum-sum-with-exactly-k-elements/maximum-sum-with-exactly-k-elements.dart
class Solution {
  int maximizeSum(List<int> nums, int k) {
    int value = nums.reduce(max) ;
    int ans = 0;
    for (int i=0;i<k;i++){
        value ++;
        ans = ans + value;
    }
    return ans - k;
  }
}

File: 2767-maximum-sum-with-exactly-k-elements/maximum-sum-with-exactly-k-elements.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var maximizeSum = function(nums, k) {
    value = Math.max(...nums)
    ans = 0
    for (let i=0;i<k;i++){
        value = value + 1
        ans = ans + value
    }
    return ans - k
};